npm install fails

Running `npm install metalsmith-sharp` from Command Prompt fails.

The cause appears to be that the installation script is trying to build Sharp v0.21.3 from source, instead of using a pre-compiled release.

Log:
```
C:\Ebackup\Blog\debigare-netlify-metalsmith>npm install metalsmith-sharp
(node:6472) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 drain listeners added to [TLSSocket]. Use emitter.setMaxListeners() to increase limit

> sharp@0.21.3 install C:\Ebackup\Blog\debigare-netlify-metalsmith\node_modules\sharp
> (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)

info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.7.0/libvips-8.7.0-win32-x64.tar.gz
info sharp Creating C:\Ebackup\Blog\debigare-netlify-metalsmith\node_modules\sharp\build\Release
info sharp Copying DLLs from C:\Ebackup\Blog\debigare-netlify-metalsmith\node_modules\sharp\vendor\lib to C:\Ebackup\Blog\debigare-netlify-metalsmith\node_modules\sharp\build\Release
prebuild-install WARN install No prebuilt binaries found (target=12.9.0 runtime=node arch=x64 libc= platform=win32)

C:\Ebackup\Blog\debigare-netlify-metalsmith\node_modules\sharp>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild)  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
gyp ERR! find Python
gyp ERR! find Python Python is not set from command line or npm configuration
gyp ERR! find Python Python is not set from environment variable PYTHON
gyp ERR! find Python checking if "python" can be used
gyp ERR! find Python - "python" is not in PATH or produced an error
gyp ERR! find Python checking if "python2" can be used
gyp ERR! find Python - "python2" is not in PATH or produced an error
gyp ERR! find Python checking if the py launcher can be used to find Python 2
gyp ERR! find Python - "py.exe" is not in PATH or produced an error
gyp ERR! find Python checking if Python is C:\Python27\python.exe
gyp ERR! find Python - "C:\Python27\python.exe" could not be run
gyp ERR! find Python checking if Python is C:\Python37\python.exe
gyp ERR! find Python - "C:\Python37\python.exe" could not be run
gyp ERR! find Python
gyp ERR! find Python **********************************************************
gyp ERR! find Python You need to install the latest version of Python 2.7.
gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
gyp ERR! find Python you can try one of the following options:
gyp ERR! find Python - Use the switch --python="C:\Path\To\python.exe"
gyp ERR! find Python   (accepted by both node-gyp and npm)
gyp ERR! find Python - Set the environment variable PYTHON
gyp ERR! find Python - Set the npm configuration variable python:
gyp ERR! find Python   npm config set python "C:\Path\To\python.exe"
gyp ERR! find Python For more information consult the documentation at:
gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
gyp ERR! find Python **********************************************************
gyp ERR! find Python
gyp ERR! configure error
gyp ERR! stack Error: Could not find any Python installation to use
gyp ERR! stack     at PythonFinder.fail (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:303:47)
gyp ERR! stack     at PythonFinder.runChecks (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:132:21)
gyp ERR! stack     at PythonFinder.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:221:16)
gyp ERR! stack     at PythonFinder.execFileCallback (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:267:16)
gyp ERR! stack     at exithandler (child_process.js:302:5)
gyp ERR! stack     at ChildProcess.errorhandler (child_process.js:314:5)
gyp ERR! stack     at ChildProcess.emit (events.js:209:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
gyp ERR! stack     at onErrorNT (internal/child_process.js:456:16)
gyp ERR! stack     at processTicksAndRejections (internal/process/task_queues.js:77:11)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Ebackup\Blog\debigare-netlify-metalsmith\node_modules\sharp
gyp ERR! node -v v12.9.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sharp@0.21.3 install: `(node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sharp@0.21.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Owner\AppData\Roaming\npm-cache\_logs\2019-09-08T21_27_15_143Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "lodash": "^4.17.4",
     "metalsmith": "^2.3.0",
     "minimatch": "^3.0.3",
-    "sharp": "^0.24.0"
+    "sharp": "^0.30.2"
   },
   "release": {
     "verifyConditions": "condition-circle"

--- a/test/index.test.js
+++ b/test/index.test.js
@@ -320,16 +320,15 @@ test('test unknown placeholders', (done) => {
 test('test catch of invalid image data error', (done) => {
   metalsmith
     .use((files) => {
-      files['example.jpg'].contents = Buffer.from('')
+      files['example.jpg'].contents = Buffer.from('this is not image data but some mocked text')
     })
     .use(sharp())
     .build((err, files) => {
       if (err) {
-        console.log(err);
         expect(
           err
             .toString()
-            .indexOf('Input Buffer is empty')
+            .indexOf('buffer contains unsupported image format')
         ).not.toBe(-1)
         return done()
       }

--- a/test/index.test.js
+++ b/test/index.test.js
@@ -328,7 +328,7 @@ test('test catch of invalid image data error', (done) => {
         expect(
           err
             .toString()
-            .indexOf('buffer contains unsupported image format')
+            .indexOf('Input buffer contains unsupported image format')
         ).not.toBe(-1)
         return done()
       }

--- a/test/index.test.js
+++ b/test/index.test.js
@@ -325,10 +325,11 @@ test('test catch of invalid image data error', (done) => {
     .use(sharp())
     .build((err, files) => {
       if (err) {
+        console.log(err);
         expect(
           err
             .toString()
-            .indexOf('Input buffer contains unsupported image format')
+            .indexOf('Input Buffer is empty')
         ).not.toBe(-1)
         return done()
       }

